test/server/http/admin_test.cc

```diff
@@ -78,7 +79,9 @@ class AdminInstanceTest : public testing::TestWithParam<Network::Address::IpVers
         cpu_profile_path_(TestEnvironment::temporaryPath("envoy.prof")),
         admin_("/dev/null", cpu_profile_path_, address_out_path_,
                Network::Test::getCanonicalLoopbackAddress(GetParam()), server_,
-               listener_scope_.createScope("listener.admin.")) {
+               listener_scope_.createScope("listener.admin.")),
+        expect_no_logs_(true) {
```

Comment:
Where is this used?

Reply:
The past.  It was used in the past :)  Removed.